client/lib/js/react/components/ledgerplugin.js

```diff
@@ -0,0 +1,142 @@
+import co from 'co/';
+
+import reconnectCore from 'reconnect-core/';
+import EventEmitter2 from 'eventemitter2/';
```

Comment:
Didn't look into it, but is this lib really needed. Browsers come with their own Event lib: https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events